conda_smithy/cli.py

```diff
@@ -257,6 +260,22 @@ def __init__(self, parser):
             action="append",
             help="drone server URL to register this repo. multiple values allowed",
         )
+        scp.add_argument(
+            "--cirun-resources",
+            action="append",
+            help="cirun resources to enable for this repo. multiple values allowed",
+        )
+        scp.add_argument(
+            "--cirun-policy-args",
```

Comment:
Where can we find a list of possible policy args?

Reply:
At this point only `pull_request` is supported, but for future reference I'll add this in the client documentation.

Reply:
Ok, then let's add a link to the docs in the help arg when it's available.

Reply:
add a link to cirun client docs at the top of `conda_smithy/cirun_utils.py`